schedule.ghactivity.php

```diff
@@ -52,6 +59,48 @@ public function record_average_label_time( $repo_name, $label ) {
 			// update_term_meta( $term_taxonomy_id[0], 'issue_slugs', $record_slugs );
 		}
 	}
+
+	public function record_projects_stats() {
+		$label_slugs = get_terms(
+			array(
+				'taxonomy'   => 'ghactivity_query_project_slug',
+				'hide_empty' => 0,
+			)
+		);
+		foreach ( $label_slugs as $term ) {
+			$term_slug = explode( '#', $term->name );
+
+			$this->record_project_stats( $term_slug[0], $term_slug[1] );
+		}
+	}
+
+	public function record_project_stats( $org_name, $project_name, $allowed_columns = null ) {
+		$record      = GHActivity_Queries::current_project_stats( $org_name, $project_name, $allowed_columns );
+		$columns     = $record[0];
+		$project_url = $record[1];
+
+		$taxonomies = array(
+			'ghactivity_query_record_type'  => 'project_stats',
+			'ghactivity_query_project_slug' => $org_name . '#' . $project_name,
+		);
+
+		$event_args = array(
+			'post_title'   => $org_name . ' | ' . $project_name . ' | ' . date( DATE_RSS ),
+			'post_type'    => 'gh_query_record',
+			'post_status'  => 'publish',
+			'tax_input'    => $taxonomies,
+			'meta_input'   => array(
+				'recorded_columns' => wp_json_encode( $columns ),
+				'project_url'      => $project_url,
+			),
+			'post_content' => 'Allowed columns: ' . wp_json_encode( $allowed_columns ),
```

Comment:
What does this display if `$allowed_columns` is set `null`? Maybe we should build the post content string separately while handing different use cases?

Reply:
The idea was to record only specific columns, but then I decided to collect all data and filter it in shortcode itself